docker/openjdk-latest/Dockerfile

```diff
@@ -1,9 +1,9 @@
 
-FROM ubuntu:22.04
+FROM ubuntu:rolling
```

Comment:
This will make reproducible builds difficult.

Reply:
what do you suggest to use for the base image?

Reply:
Fine for me, will set this to `23.10`, was using `rolling` to automatically switch to a supported version, but that could actually breaks the build, like it happened now with the missing `adduser`

Reply:
I think i can accept that tradeoff for only this build profile.

Reply:
@shemnon @jflo are you fine with the current implementation?